### PR TITLE
Add day to reminder text for 1 day call reminder

### PR DESCRIPTION
**Pull request by justinvdm**
@nathanbegbie ready for review

### CODE REVIEW

File: mentorship/mentor/models.py [hunk 1]
```diff
@@ -581,10 +582,14 @@ def notify_callee_cancelled(self):
             "Your mentoring call on %s has been cancelled" %
             display_datetime(self.call_time))
 
-    def remind_caller(self, reminder_type):
-        text = (
-            "You have a call scheduled for %s." %
-            display_time(self.call_time))
+    def remind_caller(self, reminder_type, text=None):
+        if text is None:
+            text = "You have a call scheduled for %(time)s."
```
Comment (nathanbegbie) [inline mentorship/mentor/models.py]: @justinvdm what does this syntax mean? `%(time)s` where are we getting time?

Comment (justinvdm) [inline mentorship/mentor/models.py]: It allows us to do this:

```python
>>> s = "%(foo)s %(bar)s"
>>> s % {'foo': 21, 'bar': 23}
'21 23'
>>>
```

Comment (nathanbegbie) [inline mentorship/mentor/models.py]: Cool :)

**Comment by nathanbegbie** [comment]
@justinvdm :+1:

**Pull request merged**